src/style.scss

```diff
@@ -182,3 +182,14 @@
     padding-left: unset;
   }
 }
+
+:host([disabled]) {
+  .trigger {
+    pointer-events: none;
+    color: var(--ds-color-text-ui-disabled-default, $ds-color-text-ui-disabled-default);
+  }
+
+  [auro-icon] {
+    color: var(--ds-color-icon-ui-primary-disabled-default, $ds-color-icon-ui-primary-disabled-default);
+  }
```

Comment:
auro-icon already supports a `disabled` state. We should use that instead of custom coloring within the accordion.

```
<auro-icon category="interface" name="pin-trip" disabled></auro-icon>
```